webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/header/ActivityHeadMenu.vue

```diff
@@ -70,6 +71,10 @@ export default {
         }, 200);
       }
     });
+    this.$transferRulesService.getTransfertRulesDownloadDocumentStatus().then(
+      (data) => {
+        this.downloadAllowed = data.downloadDocumentStatus === 'true';
+      });
```

Comment:
@ahmadmido please rework this PR to avoid introducing `download` term in Social, while the download buttons are defined in ecms.
 you can instead of adding here a check, adding this code snippet [here](https://github.com/exoplatform/ecms/blob/6.2.0-RC04/apps/portlet-documents/src/main/webapp/vue-app/attachment/extensions.js#L56-L81) where the download buttons are defined :


```javascript
getTransfertRulesDownloadDocumentStatus()
  .then(data => {
    if (data && data.downloadDocumentStatus === 'true') {
      extensionRegistry.registerExtension('activity', 'action', Object.assign({
        id: 'download',
        ....
      extensionRegistry.registerExtension('activity', 'action', Object.assign({
        id: 'downloadAll',
        ....
    }
  });

```